src/homework5/task2.py

```diff
@@ -0,0 +1,69 @@
+"""Создайте декоратор, который хранит результаты вызовов функции
+   (за все время вызовов, не только текущий запуск программы)
+   Задачу поместите в файл task2.py в папке src/homework5.
+"""
+
+
+import datetime
+import functools
+import logging
+import time
+
+
+def count_calls(func):
+    @functools.wraps(func)
+    def wrapper_c(*args, **kwargs):
+        wrapper_c.num_calls += 1
+        print(f"{wrapper_c.num_calls} вызов функции {func.__name__!r}")
```

Comment:
we can also add datetime and function result to logging

Reply:
we can do it in one decorator

Reply:
we have to keep all these values between program runs (here it just prints to console)